modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -217,6 +249,23 @@ export default function DatePicker({
 						yearsCheck={false}
 					/>
 
+					{!momentFormatValid && (
+						<div
+							className="error-container form-feedback-item mt-1"
+							role="alert"
+						>
+							<ClayAlert
+								className="inline-item inline-item-before"
+								displayType="danger"
+								variant="feedback"
+							>
+								{Liferay.Language.get(
+									'please-enter-a-valid-date'
+								)}
+							</ClayAlert>
+						</div>
+					)}
+
```

Comment:
hey @vngoliveira, I noticed you added a new alert here. Was this defined in any thread?

I'm looking for more information about the expected result. Since the original [thread](https://liferay.slack.com/archives/C026UR5976G/p1690907910845839) did not clearly provide the expected results.

Reply:
Sure @igor-franca! I asked for a expected result in the thread.